docs/DeveloperGuide.md

```diff
@@ -395,17 +391,31 @@ Use case ends.
 1. User !!lists contacts(UC2)!!.
 2. User requests to delete a contact.
 3. User specifies the index of the contact to delete.
-4. System deletes the contact.
+4. AB3 deletes the contact.<br>
+   Use case ends.
+
+**Extensions**
+
+<div class="step">3a. AB3 detects user input is invalid.</div>
+<div class="sub-step">3a1. AB3 displays an error message.</div>
+<div class="sub-step">3a2. User enters new input.</div>
+<div class="sub-step">Steps 3a1-3a2 are repeated until user input is valid.</div>
+<div class="sub-step">Use case resumes from step 3.</div>
```

Comment:
Should this be `Use case resumes from step 4.` instead?

Reply:
Fixed